apps/emqx_durable_storage/src/emqx_ds_storage_layer.erl

```diff
@@ -305,21 +330,21 @@ store_batch(Shard, Messages, Options) ->
 -spec prepare_batch(
     shard_id(),
     [{emqx_ds:time(), emqx_types:message()}],
-    emqx_ds:message_store_opts()
+    batch_prepare_opts()
 ) -> {ok, cooked_batch()} | ignore | emqx_ds:error(_).
-prepare_batch(Shard, Messages = [{Time, _} | _], Options) ->
+prepare_batch(Shard, Messages = [{Time, _} | _], _Options) ->
     %% NOTE
     %% We assume that batches do not span generations. Callers should enforce this.
-    ?tp(emqx_ds_storage_layer_prepare_batch, #{
-        shard => Shard, messages => Messages, options => Options
-    }),
```

Comment:
?

Reply:
This was almost exactly the same tracepoint as `emqx_ds_storage_layer_store_batch`. I transformed it into `emqx_ds_storage_layer_batch_cooked` with some extra content.

Reply:
I see, thanks. Now `store_batch` is merely a convenience wrapper that combines two separate primitives together, so I think it's better to remove `emqx_ds_storage_layer_store_batch` tracepoint and keep this one.